README.md

```diff
@@ -1,76 +1,43 @@
-# Nessie Demos
+# Nessie Binder Demos
 
-[![Main CI](https://github.com/projectnessie/nessie-demos/actions/workflows/main.yml/badge.svg)](https://github.com/projectnessie/nessie-demos/actions/workflows/main.yml)
+These demos run under binder and can be found at:
 
-The purpose of this repo is to provide a set of Jupyter notebooks (which are all unit tested) that show how Nessie can be used with other projects, such as Iceberg/Spark/Flink/Deltalake.
-Below is a list of working Jupyter notebooks, which can be run either on [Google Colab](https://colab.research.google.com/) or on [Binder](https://mybinder.org/):
+* [Spark and Iceberg](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-demo-nba.ipynb)
+* [Spark and Delta](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-delta-demo-nba.ipynb)
+* [Flink and Iceberg](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-flink-demo-nba.ipynb)
 
-* Nessie-Iceberg-NBA Jupyter Notebook
-    * [Google Colaboratory URL](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
-    * [Binder (mybinder.org)](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb)
-* Nessie-Deltalake-NBA Jupyter Notebook
-  * [Google Colaboratory URL](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-delta-demo-nba.ipynb)
-  * [Binder (mybinder.org)](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-delta-demo-nba.ipynb)
+The are automatically rebuilt every time we push to master. They are unit tested using `testbook` library to ensure we get
```

Comment:
I guess the branch here is `main` not `master` ?

Reply:
done